src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/DateValidators.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation.utils;
+
+import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+import java.time.LocalDate;
+
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.UtilsValidators.setErrorMsgToLocation;
+
+public class DateValidators {
+
+    private DateValidators() { }
+
+    public static boolean validateDateIsInPast(LocalDate compareToDate, String location, Errors errs, String loggingContext) {
```

Comment:
All these method names (which return booleans) could just start 'is', as is convention. I think it would make code more readable.

Reply:
Updated all booleans with "isValid"

Reply:
Thanks. I don't think 'Valid' is needed in the names though. Compare 'isValidDateIsInPast' to 'isDateInPast'. I think second variation reads better when inline in the code. Minor though...